synapse/storage/databases/main/devices.py

```diff
@@ -100,6 +100,7 @@ def __init__(
                 ("device_lists_outbound_pokes", "stream_id"),
                 ("device_lists_changes_in_room", "stream_id"),
                 ("device_lists_remote_pending", "stream_id"),
+                ("device_lists_changes_converted_stream_position", "stream_id"),
```

Comment:
(Not a blocking comment.)

Cross-ref, #14914 -- I really wonder if there's something systematic missing here. 😢